docs/source/02_get_started/09_end_user_faq.md

```diff
@@ -0,0 +1,63 @@
+# End User FAQ
+
+## Environments
+1. How are QHub conda user environments created? Who creates them?  
+The environment specifications are located in `qhub_config.yml` in the deployment repo. These are served to the QHub deployment using [conda-store](https://conda-store.readthedocs.io/). When you manage your environments in this way you get all of the benefits of environment versioning that QHub does under the hood, including future features such as convenient environment rollback and environment encapsulation in containers.  
+
+    Anyone with access to the QHub deployment repo can add an environment, and there are no limits to the number of environments that can be included.  
+  
+    > Do be careful of the yaml indentation as it will differ from a conda `environment.yml`.  
+
+2. I need X package and it's not in any available user environments - what do I do?  
+The proper solution is to add the package to the `qhub_config.yml` (See #1). If you don't have access to the deployment repo, you will need to contact your QHub maintainer to get the required package. Just to note: you *can* do a user install for pip packages in a pinch (this is not recommended) but they will not be available to Dask workers.
+
+3. I want to use Dask. What needs to be included in my user environment?   
+You will need to include the QHub Dask metapackage, e.g. `qhub-dask==0.2.3`. This will replace `distributed`, `dask`, ad `dask-gateway`.  
+
+4. Why can't I just create my own local conda environment or edit the existing conda environments?  
+The version of [conda-store](https://conda-store.readthedocs.io/) used in QHub versions 0.3.11 and earlier is an alpha version. This version doesn't support using local conda environments or editing pre-exising environments directly.  
+
+    See the answer to #2 for information on how to modify environments properly. In the near future the QHub team will be adding support for user-defined environments via conda-store.  
+
+4. I have a local package - how can it be installed? Will it be available to my Dask workers?  
```

Comment:
Slight numbering issue here: 4 -> 5

Reply:
ah good catch. I moved something at the last second. I'll get this fixed.